docs/document/content/test-manual/integration-test/_index.cn.md

```diff
@@ -49,7 +49,7 @@ weight = 1
 
 ## 使用指南
 
-模块路径：`shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-test-suite`
+模块路径：`shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-suite`
```

Comment:
Looks the module path doesn't exist. Is it `test/e2e/suite`?

Reply:
> Looks the module path doesn't exist. Is it `test/e2e/suite`?

I think this is ```module path```, so I wrote ```shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-suite```

Reply:
Currently, module related local directory name is different with artifactId.

I tested on Intellij IDEA "Navigate by File":
1. Navigate by `shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-suite`: No any file shown.
2. Navigate by `test/e2e/suite`: It shows `shardingsphere-test-e2e-suite` module and related files.

And also on local file system, users could locate the module by `test/e2e/suite`.

So I think use `test/e2e/suite` could benefit for users.
If `module path` is confusing, then we could consider to change these words.